packages/devtools_app/lib/src/screens/debugger/debugger_controller.dart

```diff
@@ -34,21 +34,29 @@ class DebuggerController extends DisposableController
   // in `switchToIsolate`.
   DebuggerController({
     DevToolsRouterDelegate? routerDelegate,
-    this.initialSwitchToIsolate = true,
-  }) {
+    bool initialSwitchToIsolate = true,
+  }) : _initialSwitchToIsolate = initialSwitchToIsolate {
     autoDisposeStreamSubscription(
       serviceManager.onConnectionAvailable.listen(_handleConnectionAvailable),
     );
     if (routerDelegate != null) {
       codeViewController.subscribeToRouterEvents(routerDelegate);
     }
     if (serviceManager.hasService) {
-      initialize();
+      _initialize();
     }
   }
 
   final codeViewController = CodeViewController();
 
+  late final EvalService evalService = EvalService(
```

Comment:
make this final instead of `late final`.

Reply:
Initializer cannot use getters:

<img width="715" alt="Screenshot 2022-12-27 at 2 56 24 PM" src="https://user-images.githubusercontent.com/12115586/209734039-9255805a-458e-47d7-9f42-fdd1cbca0fd7.png">

Reply:
Yeah. Move the initializer to the constructor to avoid this issue. In general, I would avoid initializing fields inline except for very trivial cases. Otherwise the initialization order semantics can trip some people up when reviewing.

Reply:
Even after the move it is still late, because it cannot be initialized in constructor initializers, but only in the body. Did i miss something? Let's follow up then.